include/linux-private/linux/if_addr.h

```diff
@@ -34,6 +34,7 @@ enum {
 	IFA_MULTICAST,
 	IFA_FLAGS,
 	IFA_RT_PRIORITY,  /* u32, priority/metric for prefix route */
+	IFA_ALIAS,
```

Comment:
these files in `include/linux-private` shall only be imported as a whole from recent kernels. They must not deviate from upstream kernel.

Also, where is `IFA_ALIAS` from? I don't find it in https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/

Reply:
_**IFA_ALIAS**_ flag does not exist in upstream kernel yet. I'm wondering now what is the policy around this kind of matters? Should we first send _IFA_ALIAS_ patch to linux kernel, and when it gets accepted, make a new pull request to libnl? If that is the case then I will close this merge-request and open it later at an appropriate time.

Reply:
Yes, first the feature needs to be added to kernel.

in particular, the kernel headers (that defines magic numbers like `IFA_ALIAS`) are the stable API of the kernel. We must not invent a different API, that is not yet accepted by kernel.

Point in case, this adds `IFA_ALIAS` with numeric value 10. In the meantime, upstream kernel added `IFA_TARGET_NETNSID` with that number.

Closing this pull request. We can only support features of the upstream, officially released kernel.